SECURITY.md

```diff
@@ -1,3 +1,19 @@
-# Security and Vulnerability Reporting
+# Security Policy
+
+## Supported Versions
+
+The pip team applies security fixes to the latest version of pip [available on PyPI](https://pypi.org/project/pip/)
```

Comment:
Um, this isn't technically true. We certainly don't backport security fixes, but we typically don't apply them to the latest version either - they simply get applied to main and then released with the next scheduled release.

I imagine that we *would* create a bugfix release for a sufficiently serious vulnerability, but it's not policy that we must - and it implies an additional commitment for the RM of a given release if we change our policy to do this (we'd need to cherry-pick the fix onto the release branch, which could be non-trivial to do if there have been other changes to the affected code in main).

Reply:
Actually, there's a further implied point here - it says we "apply security fixes", but this makes no comment on the fact that unless someone *contributes* a fix, we will not necessarily do anything. And even if someone does, we may not have the expertise to review the fix. So I'm concerned that the text as written sets an expectation that we may not be able to deliver on.

Reply:
Thanks for the review, I've created a separate issue since it seems like this will need more discussion. The primary focus of this issue is in the "Reporting a vulnerability" section and to ensure that reporters are following the correct process.